pkg/client/didexchange/client.go

```diff
@@ -113,7 +143,13 @@ func New(ctx provider) (*Client, error) {
 // CreateInvitation creates an invitation. New key pair will be generated and base58 encoded public key will be
 // used as basis for invitation. This invitation will be stored so client can cross reference this invitation during
 // did exchange protocol.
-func (c *Client) CreateInvitation(label string) (*Invitation, error) {
+func (c *Client) CreateInvitation(label string, args ...Opt) (*Invitation, error) {
+	opts := applyOptions(args...)
+
+	if len(opts.routerConnections) > 1 {
+		return nil, errors.New("only one router connection can be specified for an invitation")
```

Comment:
`opts.routerConnections` should not be an array

Reply:
@soluchok this one is pending

Reply:
I will fix it!

Reply:
@llorllale Done!